hoomd/md/methods/__init__.py

```diff
@@ -15,6 +15,21 @@
 .. rubric:: Integration methods with constraints
 
 For methods that constrain motion to a manifold see `hoomd.md.methods.rattle`.
+
+.. rubric:: Preparation
+
+Create a `hoomd.md.Integrator` to accept an integration method (or methods):
+
+.. code-block:: python
+
+    simulation = hoomd.util.make_example_simulation()
+    simulation.operations.integrator = hoomd.md.Integrator(dt=0.001)
+
+Create a `hoomd.logging.Logger` that will be used in the examples below:
```

Comment:
From a documentation user's perspective, there is no need to create the object beforehand. I would probably make this hidden given it is already quite separate from the examples using it.

Reply:
We discussed this at the hoomd development meeting and decided to hide the setup code. The minimal examples should remain minimal. They are copy and pasteable to the extent that the user has common variable names in their scripts ("simulation", "logger", etc...). If they do not have these objects already, then can find examples with a search.

The examples would all be very long if they included all setup necessary, and would then not be copy and pasteable. Copying `logger = hoomd.logging.Logger()` into your script would overwrite any existing `logger` instance you might have.